rebasehelper/build_helper.py

```diff
@@ -272,6 +272,14 @@ def _build_srpm(cls, spec, sources, patches, results_dir):
 
         return srpm, logs
 
+    @classmethod
```

Comment:
Yes it could but I make it as a classmethod cause rest of the method are classmethods as well...